From 9891bef5073b7e2d25a08b336aa5551e663db58c Mon Sep 17 00:00:00 2001
From: Philip Wilk <p.wilk@student.reading.ac.uk>
Date: Sun, 23 Mar 2025 17:31:41 +0000
Subject: [PATCH 4/4] python3Packages.mlxtend: disable failing tests
 test_ensemble_vote_classifier, test_stacking_classifier,
 test_stacking_cv_classifier

---
 pkgs/development/python-modules/mlxtend/default.nix | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/pkgs/development/python-modules/mlxtend/default.nix b/pkgs/development/python-modules/mlxtend/default.nix
index 559d54c01758..db5689580279 100644
--- a/pkgs/development/python-modules/mlxtend/default.nix
+++ b/pkgs/development/python-modules/mlxtend/default.nix
@@ -59,6 +59,10 @@ buildPythonPackage rec {
     "mlxtend/evaluate/tests/test_feature_importance.py" # urlopen error
     "mlxtend/evaluate/tests/test_bias_variance_decomp.py" # keras.api._v2
     "mlxtend/evaluate/tests/test_bootstrap_point632.py" # keras.api._v2
+    # Failing tests, most likely an upstream issue. See https://github.com/rasbt/mlxtend/issues/1117
+    "mlxtend/classifier/tests/test_ensemble_vote_classifier.py"
+    "mlxtend/classifier/tests/test_stacking_classifier.py"
+    "mlxtend/classifier/tests/test_stacking_cv_classifier.py"
   ];
 
   meta = {
-- 
2.49.0

